src/content_pr_page.tsx

```diff
@@ -58,7 +59,13 @@ async function executeScripts(instanceConfig: InstanceConfig) {
       "#repo-content-pjax-container > div > div.clearfix.js-issues-results > div.px-3.px-md-0.ml-n3.mr-n3.mx-md-0.tabnav > nav",
       "ghuibooster__spinner__large"
     );
-    await handlePrPage(instanceConfig);
+
+    const { features } = await chrome.storage.local.get("features");
```

Comment:
Was hälst du davon, wenn wir das Lesen aus dem `chrome.storage` in `getSettings` lassen?